GrepAdminBot/GrepAdminBot/Model/ConfigTemplates/ConfigurationSettings.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace GrepAdminBot.Model.ConfigTemplates
+{
+    public class ConfigurationSettings
+    {
+        public string prefix { get; set; }
```

Comment:
Is it possible to set a default ConfigurationSettings model in its class? It would look cleaner if it is possible.

Reply:
Commit be244353c9578d3fc6342ac50da7e082a8695989 should address this.

Reply:
Yep, looks good!